# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/mnemoc/ipsvd/patches/bincimap.diff
# ROCK Linux is Copyright (C) 1998 - 2004 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./service/run.in.orig	2004-05-08 12:17:59.000000000 -0400
+++ ./service/run.in	2004-05-08 12:18:44.000000000 -0400
@@ -1,11 +1,11 @@
 #!/bin/sh
 # $Id: run.in,v 1.1.1.1 2003/08/18 18:06:05 andreaha Exp $
-# daemontools supervise run-file for Binc IMAP Service.
+# ipsvd supervise run-file for Binc IMAP Service.
 
 exec 2>&1
 
-exec	tcpserver -c 100 -u 0 -g 0		\
-    -l $(hostname) -HDRP			\
+exec	tcpsvd -c 100 -u 0:0		\
+    -l $(hostname) 			\
     0 143					\
     @bindir@/bincimap-up			\
     --logtype=multilog				\
--- ./service/run-ssl.in.orig	2004-05-08 12:17:59.000000000 -0400
+++ ./service/run-ssl.in	2004-05-08 12:20:03.000000000 -0400
@@ -1,11 +1,11 @@
 #!/bin/sh
 # $Id: run-ssl.in,v 1.1.1.1 2003/08/18 18:06:05 andreaha Exp $
-# daemontools supervise run-file for Binc IMAP Service.
+# ipsvd supervise run-file for Binc IMAP Service.
 
 exec 2>&1
 
-exec	tcpserver -c 100 -u 0 -g 0			\
-    -l $(hostname) -HDRP				\
+exec	tcpsvd -c 100 -u 0:0		\
+    -l $(hostname) 				\
     0 993						\
     @prefix@/bin/bincimap-up				\
     --logtype=multilog					\
--- ./service/log/run.in.orig	2004-05-08 12:17:59.000000000 -0400
+++ ./service/log/run.in	2004-05-08 12:18:20.000000000 -0400
@@ -10,9 +10,9 @@
 # USA.
 
 # $Id: run.in,v 1.1 2003/08/23 12:53:25 andreaha Exp $
-# daemontools supervise run-file for Binc IMAP Service.
+# runit supervise run-file for Binc IMAP Service.
 
 exec 2>&1
 
-exec multilog t n5 s1048576 @localstatedir@/log/bincimap
+exec svlogd -t @localstatedir@/log/bincimap
 
--- ./service/log/run-ssl.in.orig	2004-05-08 12:17:59.000000000 -0400
+++ ./service/log/run-ssl.in	2004-05-08 12:18:20.000000000 -0400
@@ -10,9 +10,9 @@
 # USA.
 
 # $Id: run-ssl.in,v 1.1 2003/08/23 12:53:25 andreaha Exp $
-# daemontools supervise run-file for Binc IMAP Service.
+# runit supervise run-file for Binc IMAP Service.
 
 exec 2>&1
 
-exec multilog t n5 s1048576 @localstatedir@/log/bincimap-ssl
+exec svlogd -t @localstatedir@/log/bincimap-ssl
 
